src/Microsoft.CodeQuality.Analyzers/Core/QualityGuidelines/DoNotRaiseExceptionsInExceptionClauses.cs

```diff
@@ -47,21 +49,22 @@ public override void Initialize(AnalysisContext analysisContext)
             analysisContext.EnableConcurrentExecution();
             analysisContext.ConfigureGeneratedCodeAnalysis(GeneratedCodeAnalysisFlags.None);
 
-            analysisContext.RegisterOperationBlockActionInternal(operationBlockContext =>
```

Comment:
Then perhaps just surround this with `#if false` or a block comment, instead of individually commenting out all these lines.

Reply:
I'll keep the suggestion in mind for next time - I think we are good for this change as I have broken these into separate commits, which can easily be reverted once we move to newer APIs.